AK/HashTable.h

```diff
@@ -409,8 +409,17 @@ class HashTable {
     requires(IsOrdered)
     {
         VERIFY(!is_empty());
-        T element = *m_collection_data.tail->slot();
-        remove(element);
+        T element = move(*m_collection_data.tail->slot());
+        delete_bucket(*m_collection_data.tail);
+        return element;
+    }
+
+    T take_first()
```

Comment:
To be consistent, I think we should have `pop` / `shift` _or_ `take_first` / `take_last`

Reply:
I was wondering what the counterpart to `pop` was called...

Vector uses `take`, and I think that is more intuitive, so if you don't mind I will change it to that

Reply:
> so if you don't mind

It wasn't my API design to begin with so knock yourself out!